packages/babel-parser/typings/babel-parser.d.ts

```diff
@@ -106,7 +106,7 @@ export type ParserPlugin =
     'dynamicImport' |
     'estree' |
     'exportDefaultFrom' |
-    'exportNamespaceFrom' |
+    'exportNamespaceFrom' | // deprecated
```

Comment:
I wonder if a `/** @deprecated */` annotation preceding the literal would work, or if it only works with enums

Reply:
I'll check 👍